### PR TITLE
Adding the whats new entry for policy generator parameter

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/24114

### CODE REVIEW

**Review by cadawson — APPROVED** [review]
Looks good. Thanks!

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -47,6 +47,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 [#governance-whats-new]
 == Governance
 
+* You can use the `configurationPolicyAnnotations` parameter in the policy generator resource to specify key-value pair annotations on configuration policies. See link:../governance/policy_generator.adoc#policy-gen-install-on-openshift-gitops[Install the policy generator on OpenShift GitOps (ArgoCD)] for more details.
```
Comment (mprahl) [inline release_notes/whats_new.adoc]:
```suggestion
* You can use the `configurationPolicyAnnotations` parameter in the policy generator configuration to specify key-value pair annotations on generated configuration policies. See link:../governance/policy_generator.adoc#policy-gen-install-on-openshift-gitops[Install the policy generator on OpenShift GitOps (ArgoCD)] for more details.
```

Comment (mprahl) [inline release_notes/whats_new.adoc]: The link is not correct.

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -47,6 +47,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 [#governance-whats-new]
 == Governance
 
+* You can use the `configurationPolicyAnnotations` parameter in the policy generator configuration to specify key-value pair annotations on generated configuration policies. See link:../governance/policy_generator.adoc#policy-gen-install-on-openshift-gitops[Install the policy generator on OpenShift GitOps (ArgoCD)] for more details.
```
Comment (mprahl) [inline release_notes/whats_new.adoc]: The link is to installing the Policy Generator on OpenShift GitOps and not about the new parameter.

Comment (dockerymick) [reply release_notes/whats_new.adoc]: Thanks!

**Review by mprahl — APPROVED** [review]

**Pull request merged**